app/v1/models/request/amendPeriodic/PeriodDisallowableExpenses.scala

```diff
@@ -0,0 +1,76 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.models.request.amendPeriodic
+
+import play.api.libs.functional.syntax._
+import play.api.libs.json._
+
+case class PeriodDisallowableExpenses(costOfGoodsDisallowable: Option[BigDecimal],
+                                      paymentsToSubcontractorsDisallowable: Option[BigDecimal],
+                                      wagesAndStaffCostsDisallowable: Option[BigDecimal],
+                                      carVanTravelExpensesDisallowable: Option[BigDecimal],
+                                      premisesRunningCostsDisallowable: Option[BigDecimal],
+                                      maintenanceCostsDisallowable: Option[BigDecimal],
+                                      adminCostsDisallowable: Option[BigDecimal],
+                                      businessEntertainmentCostsDisallowable: Option[BigDecimal],
+                                      advertisingCostsDisallowable: Option[BigDecimal],
+                                      interestOnBankOtherLoansDisallowable: Option[BigDecimal],
+                                      financeChargesDisallowable: Option[BigDecimal],
+                                      irrecoverableDebtsDisallowable: Option[BigDecimal],
+                                      professionalFeesDisallowable: Option[BigDecimal],
+                                      depreciationDisallowable: Option[BigDecimal],
+                                      otherExpensesDisallowable: Option[BigDecimal])
+
+object PeriodDisallowableExpenses {
+
+  implicit val reads: Reads[PeriodDisallowableExpenses] = (
+    (JsPath \ "costOfGoodsDisallowable").readNullable[BigDecimal] and
+      (JsPath \ "paymentsToSubcontractorsDisallowable").readNullable[BigDecimal] and
+      (JsPath \ "wagesAndStaffCostsDisallowable").readNullable[BigDecimal] and
+      (JsPath \ "carVanTravelExpensesDisallowable").readNullable[BigDecimal] and
+      (JsPath \ "premisesRunningCostsDisallowable").readNullable[BigDecimal] and
+      (JsPath \ "maintenanceCostsDisallowable").readNullable[BigDecimal] and
+      (JsPath \ "adminCostsDisallowable").readNullable[BigDecimal] and
+      (JsPath \ "businessEntertainmentCostsDisallowable").readNullable[BigDecimal] and
+      (JsPath \ "advertisingCostsDisallowable").readNullable[BigDecimal] and
+      (JsPath \ "interestOnBankOtherLoansDisallowable").readNullable[BigDecimal] and
+      (JsPath \ "financeChargesDisallowable").readNullable[BigDecimal] and
+      (JsPath \ "irrecoverableDebtsDisallowable").readNullable[BigDecimal] and
+      (JsPath \ "professionalFeesDisallowable").readNullable[BigDecimal] and
+      (JsPath \ "depreciationDisallowable").readNullable[BigDecimal] and
+      (JsPath \ "otherExpensesDisallowable").readNullable[BigDecimal]
+    )(PeriodDisallowableExpenses.apply _)
```

Comment:
these reads can just be generic json reads since there is no field name transformations